includes/widgets/google-maps.php

```diff
@@ -108,14 +108,26 @@ protected function register_controls() {
 			$api_key = get_option( 'elementor_google_maps_api_key' );
 
 			if ( ! $api_key ) {
+				$link_template = '<a href="%1$s" target="_blank">%2$s</a>';
+				$integration_settings_link = sprintf(
+					$link_template,
+					Settings::get_url() . '#tab-integrations',
+					esc_html__( 'Integrations Settings', 'elementor' )
+				);
+				$create_key_link = sprintf(
+					$link_template,
+					'https://developers.google.com/maps/documentation/embed/get-api-key',
+					esc_html__( 'here.', 'elementor' )
+				);
+
 				$this->add_control(
 					'api_key_notification',
 					[
 						'type' => Controls_Manager::RAW_HTML,
 						'raw' => sprintf(
-							esc_html__( 'Set your Google Maps API Key in Elementor\'s <a href="%1$s" target="_blank">Integrations Settings</a> page. Create your key <a href="%2$s" target="_blank">here.', 'elementor' ),
-							Settings::get_url() . '#tab-integrations',
-							'https://developers.google.com/maps/documentation/embed/get-api-key'
+							esc_html__( 'Set your Google Maps API Key in Elementor\'s %1$s page. Create your key %2$s', 'elementor' ),
```

Comment:
It's better to make translatable text in one paragraph and not separate them into many parts, please look at [this example](https://github.com/elementor/elementor/blob/6d61615d9b4cc9a5d659da19ab9cdd5bd6ca9f35/core/experiments/manager.php#L424) and add also the "Translators" comment

Reply:
@ItzhakAvm Please fix like the example above (with the "translators" comment, and the HTML as a translation parameter)